File: Q8.cs
using System;

namespace Q8
{
    class Program
    {
        static void Main(string[] args)
        {
            ushort[] idade = new ushort[20];
            ushort soma = 0;
            byte vezes = 1;
            for (byte i = 0; i < idade.Length; i ++, vezes ++)
            {
                Console.Write("Digite a {0}ª idade: ", vezes);
                idade[i] = Convert.ToUInt16(ushort.Parse(Console.ReadLine()));
                soma = Convert.ToUInt16(soma + idade[i]);
            }
            Console.WriteLine("A soma das 20 idades que você digitou é {0} anos.", soma);
        }
    }
}

File: Q8.java
import java.util.Scanner;

public class Q8
{
    public static Scanner leia = new Scanner(System.in);
    public static void main(String args[])
    {
        int idade[] = new int[20], soma = 0, vezes = 1;
        for (int i = 0; i < idade.length; i ++, vezes ++)
        {
            System.out.printf("Digite a %dª idade: ", vezes);
            idade[i] = leia.nextInt();
            soma = soma + idade[i];
        }
        System.out.printf("A soma das 20 idades que você digitou é %d anos.\n", soma);
    }
}